apps/admin-ui/src/user/UsersTabs.tsx

```diff
@@ -85,17 +86,23 @@ const UsersTabs = () => {
     setAddedGroups(groups);
   };
 
-  const save = async (user: UserRepresentation) => {
-    user.username = user.username?.trim();
+  const save = async (formUser: UserRepresentation) => {
+    formUser.username = formUser.username?.trim();
 
     try {
       if (id) {
-        await adminClient.users.update({ id }, user);
+        await adminClient.users.update(
+          { id },
+          {
+            ...formUser,
+            attributes: { ...user?.attributes, ...formUser.attributes },
+          }
+        );
         addAlert(t("userSaved"), AlertVariant.success);
         refresh();
       } else {
-        user.groups = addedGroups.map((group) => group.path!);
-        const createdUser = await adminClient.users.create(user);
+        formUser.groups = addedGroups.map((group) => group.path!);
+        const createdUser = await adminClient.users.create(formUser);
```

Comment:
Instead of mutating `formUser` (which might have unintended side-effects), you could spread it into the call directly.  It's not used anywhere else, so in-lining it like this is fine.
```suggestion
        const createdUser = await adminClient.users.create({
          ...formUser,
          groups: addedGroups.map((group) => group.path!)
        });
```